ibm/mas_devops/roles/dro/tasks/uninstall/main.yml

```diff
@@ -0,0 +1,129 @@
+---
+# 1. Delete CR, CSV and Subscription
+# -----------------------------------------------------------------------------
+# Delete the MarketplaceConfig
+- name: "uninstall : Delete the MarketplaceConfig"
+  kubernetes.core.k8s:
+    state: absent
+    api_version: marketplace.redhat.com/v1alpha1
+    kind: MarketplaceConfig
+    namespace: redhat-marketplace
+    name: marketplaceconfig
+    wait: yes
+    wait_timeout: 600
+
+# IMO
+- name: "uninstall : Delete ibm-metrics-operator CSV"
+  kubernetes.core.k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: ClusterServiceVersion
+    namespace: redhat-marketplace
+    label_selectors:
+      - "operators.coreos.com/ibm-metrics-operator.redhat-marketplace"
+    wait: yes
+    wait_timeout: 300
+
+- name: "uninstall : Delete ibm-metrics-operator Subscription"
+  kubernetes.core.k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: Subscription
+    name: ibm-metrics-operator
+    namespace: redhat-marketplace
+    wait: yes
+    wait_timeout: 300
+
+# DRO
+- name: "uninstall : Delete  ibm-data-reporter-operator CSV"
+  kubernetes.core.k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: ClusterServiceVersion
+    namespace: redhat-marketplace
+    label_selectors:
+      - "operators.coreos.com/ibm-data-reporter-operator.redhat-marketplace"
+    wait: yes
+    wait_timeout: 300
+
+- name: "uninstall : Delete ibm-data-reporter-operator Subscription"
+  kubernetes.core.k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: Subscription
+    name: ibm-data-reporter-operator
+    namespace: redhat-marketplace
+    wait: yes
+    wait_timeout: 300
+
+
+# RMO
+- name: "uninstall : Delete  redhat-marketplace-operator CSV"
+  kubernetes.core.k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: ClusterServiceVersion
+    namespace: redhat-marketplace
+    label_selectors:
+      - "operators.coreos.com/redhat-marketplace-operator.redhat-marketplace"
+    wait: yes
+    wait_timeout: 300
+
+- name: "uninstall : Delete redhat-marketplace-operator Subscription"
+  kubernetes.core.k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: Subscription
+    name: redhat-marketplace-operator
+    namespace: redhat-marketplace
+    wait: yes
+    wait_timeout: 300
+
+# Delete CRs
+- name: "uninstall : Delete the MarketplaceConfig"
```

Comment:
seems to be a duplicate of the first task

Reply:
removed duplicate tasks